docker/test/opensuse.Dockerfile

```diff
@@ -14,7 +14,8 @@ ARG pip_install_flags="--user"
 ARG preinstalled_modules="numpy pytest nvidia-cublas-cu11"
 
 ARG DEBIAN_FRONTEND=noninteractive
-RUN zypper --non-interactive up --no-recommends && zypper --non-interactive in --no-recommends \
+RUN zypper clean --all && zypper --non-interactive up --no-recommends \
+    && zypper --non-interactive in --no-recommends \
```

Comment:
Was this change intended to fix this error: https://github.com/NVIDIA/cuda-quantum/actions/runs/6865626131/job/18670804127?pr=922 ? If so, it may not have 100% fixed it. That being said - it doesn't seem like this change would have any harm, so it seems file to keep it.

Reply:
Yes, and I know. Now it is a different error...